.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Release checklist
+about: Použi tento checklist na testovanie artefaktov pred public releaseom
+title: "Release "
+labels: release
+assignees: @slovensko-digital/autogram-release-team
+---
+
+## Windows
+
+- [ ] funguje inštalácia na Windows cez stiahnutý .msi
+- [ ] funguje spustenie v GUI móde
+- [ ] funguje URL handler [autogram://go](autogram://go)
+- [ ] funguje GUI otvoriť jeden súbor, ten sa zobrazí, viem ho podpísať, vytvorí sa podpísaný súbor
```

Comment:
Tuto by som skusal cez /docs viac typov suborov, mozno @celuchmarek vie identifikovat take problematicke. Ja si pamatam, ze problem bol naposledy s tym xslt na velke subory a to by toto neodhalilo.